src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/TypesThatOwnDisposableFieldsShouldBeDisposable.cs

```diff
@@ -60,7 +60,9 @@ protected abstract class DisposableFieldAnalyzer
         {
             private readonly DisposeAnalysisHelper _disposeAnalysisHelper;
 
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
```

Comment:
❔ How is this a false positive? Can you not use `RosylnDebug.Assert`?

Reply:
Thanks, I was expecting Debug.Assert to take care of it. Changed now.